x-pack/plugins/triggers_actions_ui/public/application/hooks/use_bulk_operation_toast.tsx

```diff
@@ -135,10 +136,10 @@ export const useBulkOperationToast = ({
           SINGLE_RULE_TITLE,
           MULTIPLE_RULE_TITLE
         ),
-        text: toMountPoint(renderToastErrorBody(errors, 'warning'), { i18n, theme }),
+        text: toMountPoint(renderToastErrorBody(errors, 'warning'), { i18n, theme, userProfile }),
```

Comment:
Not passing a large object here for the services, because this is within a `useCallback`. See https://github.com/elastic/kibana/pull/202606#discussion_r1871588439